src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.jelly

```diff
@@ -0,0 +1,204 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:i="jelly:fmt" xmlns:l="/lib/layout" xmlns:st="jelly:stapler">
+    <div class="custom-pane-frame-padding">
+        <j:set var="hashAndMetadataList" value="${it.getApprovedFullScriptMetadata()}" />
+        <h2>
+            ${%approvedScripts}
+        </h2>
+        
+        <div id="already-approved">
+            <j:if test="${!it.isMetadataGatheringEnabled()}">
+                <div class="alert alert-info">
+                    ${%metadataGatheringDisabled}
+                </div>
+            </j:if>
+            <j:choose>
+                <j:when test="${hashAndMetadataList.isEmpty()}">
+                    <p class="no-approved-scripts">
+                        <i>${%noApprovedScripts}</i>
+                    </p>
+                </j:when>
+                <j:otherwise>
+                    <table class="pane bigtable">
+                        <tr>
+                            <th width="1%"><!-- checkbox --></th>
+                            <th tooltip="${%approvedHash_tooltip}" class="info-cursor">${%approvedHash}</th>
```

Comment:
Why does the user care about the hash?

Reply:
In the current version only the hash is stored in the XML. By providing it in the table display I initially thought it could be good for the user to know which hash correspond to what scripts. The metadata and the hashes are separated.
But as a second thought with your question, I am no longer certain it's a good idea :p 

Potentially hidden the hash as just a HTML attribute could be sufficient for advance users and keeping the UI simpler... good point :)

Reply:
Yeah not sure, the only reason I can see it for a user as an ID.
I would lean towards less is more though

Reply:
Removed